cmd/veneur/main.go

```diff
@@ -187,6 +189,12 @@ func main() {
 	if err != nil {
 		logger.WithError(err).Fatal("Could not initialize server")
 	}
+
+	for range time.Tick(server.Interval) {
+		go diagnostics.CollectUptimeMetrics(server)
+		go diagnostics.CollectRuntimeMemStats(server)
+	}
```

Comment:
Nit: move this inside the `diagnostics` package.

Reply:
move for loop to a `CollectDiagnosticsMetrics` function in `diagnostics `package